gammapy/visualization/utils.py

```diff
@@ -126,9 +130,15 @@ def patch_func(
 
 def plot_contour_line(ax, x, y, **kwargs):
     """Plot smooth curve from contour points"""
+    xf = x
+    yf = y
+
     # close contour
-    xf = np.append(x, x[0])
-    yf = np.append(y, y[0])
+    rel_tol = 1.e-2
+    if not (math.isclose(x[0], x[-1], rel_tol=rel_tol) and math.isclose(y[0], y[-1], rel_tol=rel_tol)):
+        xf = np.append(x, x[0])
+        yf = np.append(y, y[0])
+        log.info("Closing the contours.")
```

Comment:
Is logging really useful here? It does not really help users. I would remove.

Reply:
As suggested, a simple test of equality is used and the logging is removed.